### ISSUE TITLE
Deploy aggregator to the cloud

### ISSUE THREAD

**Issue by upkarlidder** [Call-for-Code-for-Racial-Justice/Open-Sentencing-Aggregator#23]

**Comment by upkarlidder** [comment]
I was able to deploy the current aggregator to Code Engine

https://open-sentencing-aggregator-app.deoblc6jtd1.us-south.codeengine.appdomain.cloud/openapi/ui/

The code is not complete and the application might need a re-write as @RickPoleshuck pointed out. However, still wanted to get what we have to the cloud. Will put a CI/CD in place so new code gets deployed automatically after going through linters and other build checks.

**Comment by RickPoleshuck** [comment]
Agreed. With greenfield projects, there is always a lot of work that
has to be redone. Redeploying a revised aggregator will be easier after
we do it once.
